### ISSUE TITLE
Hessian2 Deserializer Joda-time error

### ISSUE THREAD

**Issue by Bricks-Man** [dubbo/hessian-lite#6]
我在dubbo项目上看到了一个[issue](https://github.com/apache/incubator-dubbo/issues/126)。我跟踪了一下代码发现是你们这个hessian工程导致的。我在Hessian 4.0.38上没有重新该问题。经过debug，我发现是因为你们的代码缺少了UnsafeDeserializer.class相关类导致的，我在sofa上看到他们已经把hessian协议升级到了高版本。你们这边有升级的计划吗？https://github.com/alipay/sofa-hessian

**Comment by lovepoem** [comment]
>我发现是因为你们的代码缺少了UnsafeDeserializer.class相关类导致的，

Yes, the hessian-lite now have this issue, it seems that not easy to upgrade the latest version
I will try to fix it on the codebase of hessian-lite

**Comment by lovepoem** [comment]
refer https://github.com/apache/incubator-dubbo/issues/126

**Comment by hhls** [comment]
> > 我发现是因为你们的代码缺少了UnsafeDeserializer.class相关类导致的，
> 
> Yes, the hessian-lite now have this issue, it seems that not easy to upgrade the latest version
> I will try to fix it on the codebase of hessian-lite

Is it good now？

**Comment by ToryZhou** [comment]
现在这个问题修复了么？我现在dubbo3.0.4版本还是有这个问题。我在想除了jodatime的反序列化有问题，其他类型的反序列化有没有问题，先把jodatime这个要想法弄下啊。
